src/ResourceManager/Sql/Commands.Sql/Database/Model/DatabaseCreateMode.cs

```diff
@@ -0,0 +1,75 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.Management.Sql.Models
```

Comment:
There should already be a CreateMode enum in the SDK, can that be used instead?

Reply:
@jaredmoo yes, found that we have it in the SDK. will use the one in SDK instead, remove this redundant CreateMode class.